static/monitor.js

```diff
@@ -65,7 +65,12 @@
             yAxis: {
                 startOnTick: true,
                 endOnTick: true,
-                title: null
+                title: null,
+                plotBands: [{
```

Comment:
for now, let's leave these in, but longer-term we should figure out how to make them configurable.